counsel.el

```diff
@@ -1509,6 +1509,7 @@ When CMD is a string, use it as a \"git grep\" command.
 When CMD is non-nil, prompt for a specific \"git grep\" command."
   (interactive)
   (let ((proj-and-cmd (counsel--git-grep-cmd-and-proj cmd))
+        (enable-recursive-minibuffers t)
```

Comment:
I think `counsel--git-grep-cmd-and-proj` can use the minibuffer, so maybe `enable-recursive-minibuffers` should come first?

Reply:
Good point, I'll move it up.  Thanks.